css/comp_lib_styles/components/snackbar.css

```diff
@@ -0,0 +1,43 @@
+.snackbar {
+    position: fixed;
+    right: 1rem;
+    bottom: 30px;
+    min-width: 250px;
+    padding: 1rem;
+    display: none;
+    z-index: var(--z-index-five);
+    border-radius: var(--border-radius-sm);
+    color: var(--clr-white-full);
+    background-color: var(--clr-black-full);
+}
+
+.snackbar .snackbar-content {
+    margin: 0 0.5rem 0 0;
```

Comment:
Again, can use px here since it's generally not desirable to scale very small values with rem.